src/skate_edges.py

```diff
@@ -0,0 +1,23 @@
+#Test Footage For Class
+
+import cv2
+import numpy as np
+
+cap = cv2.VideoCapture(1)
+
+while(cap.isOpened()):
+    ret, frame = cap.read()
+
+    vid = cv2.cvtColor(frame, cv2.COLOR_BGR2GRAY)
+    
+    ret,th = cv2.threshold(vid,100,200,cv2.THRESH_BINARY)
```

Comment:
can we get spacing, such as:
```suggestion
    ret, th = cv2.threshold(vid, 100, 200, cv2.THRESH_BINARY)
```

Reply:
cleaner code imo, also good to be consistent